runtime/common/src/constants.rs

```diff
@@ -121,8 +121,7 @@ pub type HandleSuffixMax = ConstU16<99>;
 // -end- Handles Pallet
 
 // --- TimeRelease Pallet ---
-// Update
-pub type MinReleaseTransfer = ConstU128<0>;
+pub type MinReleaseTransfer = ConstU128<{ currency::EXISTENTIAL_DEPOSIT }>;
```

Comment:
Hmm, could this be problematic?
I can understand that we might not want to schedule a transfer that might result in an underfunded account that will just get reaped, but won't this also prevent scheduling potentially legitimate transfers to accounts that already hold enough tokens to not be reaped?

Or is this sort of a "lesser of two evils" situation, ie yes, it disallows some minimal (and therefore unlikely?) scheduled transfer amounts, but guarantees that (at least at the time of scheduling) we won't schedule a transfer that will end up getting reaped?

Or, am I misunderstanding the purpose of this value?

Reply:
This amount is very small, just one cent. We can reasonably expect users to send transfers larger than that. The main purpose of this limit is to help prevent storage spam.